frontend/src/org/components/EventForm/steps/RegistrationStep.module.scss

```diff
@@ -1,81 +1,6 @@
 @use 'styles/colors';
 @use 'styles/media';
 
-.questionsBox {
```

Comment:
Zkoušel jsem z toho udělat separé komponentu (podobně viz `RegistrationStep.tsx`), aby se dala použít i ve vytváření zpětnovazebního formuláře, strašně jsem se teda u toho posekal na typech, takže jsem to možná měl raději zkopírovat, no, co už. :)